cs-couchbase/src/main/java/io/cloudslang/content/couchbase/factory/HeadersBuilder.java

```diff
@@ -0,0 +1,37 @@
+/*******************************************************************************
+ * (c) Copyright 2017 Hewlett-Packard Development Company, L.P.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License v2.0 which accompany this distribution.
+ *
+ * The Apache License is available at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *******************************************************************************/
+package io.cloudslang.content.couchbase.factory;
+
+import io.cloudslang.content.couchbase.entities.inputs.InputsWrapper;
+
+import static io.cloudslang.content.couchbase.entities.constants.Constants.Api.BUCKETS;
+import static io.cloudslang.content.couchbase.factory.buckets.BucketHeadersBuilder.setBucketHeaders;
+
+/**
+ * Created by TusaM
+ * 4/14/2017.
+ */
+public class HeadersBuilder {
+    private static final String UNKNOWN_COUCHBASE_HEADER = "Unknown Couchbase header.";
```

Comment:
why don't you add this in the Constants class 
(cs-couchbase/src/main/java/io/cloudslang/content/couchbase/entities/constants/Constants.java)
or somewhere more fit ?